examples/busybox/snapcraft.yaml

```diff
@@ -0,0 +1,34 @@
+name: busybox
+version: 1.0
+summary: busybox example to showcase the snapcraft kbuild plugin
+description:
+   this is a snapcraft example package that contains the output of the
+   snapcraft busybox example.
+
+apps:
+  ls:
+    command: ls
+    plugs: [old-security]
```

Comment:
I was avoiding writing `interface: old-security` in case that changed again :smile: 

I don't mind otherwise :wink: